src/pseudopeople/configuration/noise_configuration.py

```diff
@@ -31,7 +32,7 @@ def get_value(
         noise_type: str,
         parameter_name: str,
         column_name: Optional[str] = None,
-    ) -> Union[float, int, dict]:
+    ) -> Union[float, int, list, dict]:
```

Comment:
nit: I think we're trying to start using update type hinting on at least lines we add/modify (list and dict sub-types, using | instead of Union, etc)